OSDOCS-3510: Windows Server 2022 support in WINC 6.0.0 (#48535)
openshift/openshift-docs: modules/creating-the-vsphere-windows-vm-golden-image.adoc

.Procedure

.  Select a compatible Windows Server version. Currently, the Windows Machine Config Operator (WMCO) stable version supports Windows Server 2022 Long-Term Servicing Channel with the OS-level container networking patch link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[KB5012637].

> . Create a new VM in the vSphere client using the VM golden image with a compatible Windows Server version, as described in the *Windows Machine Config Operator prerequisites* section of the  *Red Hat OpenShift support for Windows Containers release notes*.

Reviewer: The Red Hat Openshift support for Windows Containers release notes is an assembly. Is there a reason why you chose not to link to the specific section in the release notes?

Reviewer: I am always confused by this @mburke5678. I think you are correct here. We cannot create links from modules. Sorry.

Reviewer: I cannot find anything definitive, but I have seen quotes (instead of bold) used frequently when having to use a static xref. I suggest quotes.

Author: I believe you are correct.

> . Create a new VM in the vSphere client using the VM golden image with a compatible Windows Server version, as described in the *Windows Machine Config Operator prerequisites* section of the  *Red Hat OpenShift support for Windows Containers release notes*.

Reviewer: ```suggestion
. Create a new VM in the vSphere client using the VM golden image with a compatible Windows Server version. For more information about compatible versions, see the *Windows Machine Config Operator prerequisites* section of the  *Red Hat OpenShift support for Windows Containers release notes*.
```
For inline links, the correct format is "For more information about..." [1]

[1] https://docs.engineering.redhat.com/display/CCSENABLE/Inline+links
